docker-entrypoint.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# If long & short hostnames are not the same, use long hostnames
+if ! [[ $(hostname) == $(hostname -s) ]]; then
+	export RABBITMQ_USE_LONGNAME=true
+fi
+
+if ! [[ -z ${RABBIT_SSL_CERT_FILE+x} ]] || ! [[ -z ${RABBIT_SSL_KEY_FILE+x} ]] || ! [[ -z ${RABBIT_SSL_CA_FILE+x} ]]; then
```

Comment:
A bit of  nitpicking but this test looks overcomplicated, especially with the double negation (`!` + `-z`). It would be more logical to me to use AND operators instead of OR + negation.

I'm not used to Bash's `[[ ]]` extension, but with plain Bourne shell, you could use:
```sh
# "-a" is AND; see test(1).
if [ ${RABBIT_SSL_CERT_FILE+x} -a ${RABBIT_SSL_KEY_FILE+x} -a ${RABBIT_SSL_CA_FILE+x} ]; the
```

Speaking about Bash extensions, the script mixes tests using Bourne shell and Bash's extension. I believe you could get rid of all uses of `[[ ]]` to use `[ ]`:
```sh
if [ $(hostname) != $(hostname -s) ]; then
```
```sh
if [ "${use_ssl}" = "yes" ]; then
```